internal/log/plain.go

```diff
@@ -21,7 +21,9 @@ var _ Sink = (*plainSink)(nil)
 
 func newPlainSink(w io.Writer, logTime bool) *plainSink {
 	var isaTTY bool
-	if f, ok := w.(*os.File); ok {
+	if os.Getenv("FTL_LOG_COLOR") != "" {
```

Comment:
Make this a Kong flag in `log.Config`, and pass it down into this function to avoid global state.

Reply:
I implemented this, but my main reason for this feature is in running tests and it now doesn't work with tests because `ContextWithNewDefaultLogger` doesn't go through `log.Config`. I did make a env check in there which is only checked when running tests.

Reply:
Makes sense 👍